cocotb/scheduler.py

```diff
@@ -67,7 +50,19 @@
 from cocotb.log import SimLog
 from cocotb.result import TestComplete
 from cocotb.utils import remove_traceback_frames
-from cocotb import _py_compat
+from cocotb import outcomes,_py_compat
```

Comment:
nit:
```suggestion
from cocotb import outcomes, _py_compat
```

Reply:
Sharp eyes :)

Reply:
I'm interested that flake8 didn't pick that up

Reply:
@ktbarrett
https://github.com/cocotb/cocotb/blob/43adc60479c193d9dc700c3851c7db2907034e35/setup.cfg#L17